Source/ChocolateyGui/ViewModels/SettingsViewModel.cs

```diff
@@ -270,6 +272,26 @@ public void Back()
 
             var sources = await _packageService.GetSources();
             Sources.AddRange(sources);
+        }
+
+        private void LocalizateFeaturesDescription(ChocolateyFeature feture)
+        {
+            var key = string.Format("SettingsViewModel_FeatureDescription_{0}", feture.Name);
+            var description = Resources.ResourceManager.GetString(key, Resources.Culture);
+            if (!string.IsNullOrWhiteSpace(description))
+            {
+                feture.Description = description;

```

Comment:
@gep13 talking CI - could we implement something that would cause an Warning/Error on build? - this way we could at least ensure all settings strings were available/translated on build date?